pkg/client/session.go

```diff
@@ -14,6 +14,10 @@ import (
 	"google.golang.org/grpc"
 )
 
+// OpenSession establishes a new session on a server, this method also opens new
```

Comment:
OpenSession establishes a new session with the server. wdyt?

Reply:
Changed in d0cfa1c9